### PR TITLE
added module_repository parameter

### PR DESCRIPTION
**Pull request by koweblomke**
enable using a different puppet module repository

### CODE REVIEW

**Comment by aamerik** [comment]
Any chance this can be merged?  I need to override the default forge repo URL. Thanks!

**Comment by mmoll** [comment]
sorry for that sitting so long. :(
please rebase

File: manifests/params.pp [hunk 1]
```diff
@@ -17,6 +17,7 @@
   $cron_cmd            = undef
   $agent_noop          = false
   $show_diff           = false
+  $module_repository   = ''
```
Comment (mmoll) [inline manifests/params.pp]: ```undef``` please

**Comment by mmoll** [comment]
a test would also be nice

**Comment by mmoll** [comment]
continuation in #226

**Pull request closed without merging**